app/Http/Controllers/PostsController.php

```diff
@@ -80,6 +80,9 @@ public function index(Request $request)
         // Only return posts tagged "Hide In Gallery" if staff user or if is owner of the post.
         $query = $query->withHiddenPosts();
 
+        // Do not return any voter-reg posts.
+        $query = $query->where('type', '!=', 'voter-reg');
```

Comment:
@DoSomething/team-bleed we probably want to hide this behind a query param, correct? 

I'm thinking that we want the API to expose posts that have the `type` `voter-reg` but Rogue will use the query param to hide these posts in Rogue Admin?